code/game/objects/items/circuitboards/computer_circuitboards.dm

```diff
@@ -391,15 +391,15 @@
 
 //Security
 
-/obj/item/circuitboard/computer/gulag_teleporter_console
+/obj/item/circuitboard/computer/gulag_teleporter_console //NSV13 modified - fix
 	name = "labor camp teleporter console (Computer Board)"
 	icon_state = "security"
-	build_path = /obj/machinery/computer/prisoner/gulag_teleporter_computer
+	build_path = /obj/item/circuitboard/computer/gulag_teleporter_console
```

Comment:
This builds... itself?

Reply:
Yeah honestly I have no clue what the fuck happened there.

Reply:
> This builds... itself?

There's also the one for the machine part... but I couldn't find it.